_includes/layout/seo-meta.html

```diff
@@ -45,19 +45,19 @@
         <meta property="og:image" content="{{ site.url }}{{ page.images.first }}" />
         <meta name="twitter:image" content="{{ site.url }}{{ page.images.first }}" />
     {% else %}
-        <meta property="og:image" content="{{ site.url }}/img/ritlug-large.png" />
-        <meta name="twitter:image" content="{{ site.url }}/img/ritlug-large.png" />
+        <meta property="og:image" content="{{ site.url }}/img/logo.png" />
+        <meta name="twitter:image" content="{{ site.url }}/img/logo.png" />
     {% endif %}
     <script type="application/ld+json">
     {
         "@context": "http://schema.org",
         "@type": "Organization",
-        "name": "RIT Linux Users Group",
-        "alternateName": "RITlug",
-        "logo": "{{ site.url }}/img/ritlug-large.png",
+        "name": "{{ site.title }}",
+        "alternateName": "{{ site.title-abbrev }}",
+        "logo": "{{ site.url }}/img/logo.png",
         "url": "{{ site.url }}",
         "sameAs": [
-            "https://twitter.com/RITlug"
+            "https://twitter.com/{{ site.twitter_account }}"
         ],
         "knowsAbout": [
```

Comment:
You might want to update this too

Reply:
@ct-martin Fair point. I'll work on adding in different keywords in a future commit. It would probably be good to set these as a list in the config too. I'll see if I can figure it out.